proto/viam/app/v1/robot.proto

```diff
@@ -81,6 +81,7 @@ message ServiceConfig {
   string namespace = 2;
   string type = 3;
   google.protobuf.Struct attributes = 4;
+  repeated string depends_on = 5;
```

Comment:
we need to add models here too - maybe add it in this PR and then make a separate ticket for the associated changes in rdk/app